src/trufont/objects/factories.py

```diff
@@ -97,6 +100,36 @@ def pathGraphicsPathFactory(path):
             if point.type == "curve":
                 graphicsPath.AddCurveToPoint(*stack)
                 stack = []
+            elif point.type == "qcurve" or (not open_ and point is points[-1]):
+                assert len(stack) % 2 == 0, "Stack must have even number of points."
```

Comment:
Comment is basically the same thing as the code before, so unneeded. If the assert fails, the LOC will be included in the traceback either way.

Reply:
You only need to assert this in the else below